### PR TITLE
Mria replicant manual join

### PR DESCRIPTION
**Pull request by ieQu1**
Start `emqx_ctl` process before `emqx_machine` application, to get the CLI working, even if the rest of the EMQX is down. It may happen when on replicant node when it can't join to any core node.

Sadly, because emqx bootstrapping is done in a separate application, and because `apps/emqx` may be used standalone, I had no other way, but to split emqx_ctl module to a separate app to make all static checks happy. This is, admittedly, a pretty hacky solution, since this new repo is useless on its own.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -88,7 +88,8 @@ defmodule EMQXUmbrella.MixProject do
       {:ranch,
        github: "ninenines/ranch", ref: "a692f44567034dacf5efcaa24a24183788594eb7", override: true},
       # in conflict by grpc and eetcd
-      {:gpb, "4.19.5", override: true, runtime: false}
+      {:gpb, "4.19.5", override: true, runtime: false},
+      {:emqx_ctl, github: "emqx/emqx_ctl", tag: "0.1.0", override: true}
```
Comment (zmstone) [inline mix.exs:92]: does it have to be an external dependency ?
should it work if we add a `apps/emqx_ctl` ?

Comment (zmstone) [reply mix.exs:92]: emqx app should not need to depend on emqx_ctl?

**Comment by zmstone** [comment]
apps/emqx/test/emqx_ctl_SUITE.erl should be moved?

File: mix.exs (hunk 1 above)
Comment (ieQu1) [reply mix.exs:92]: emqx app registers some CLI commands, so it does depend on it.

**Review by zmstone — APPROVED** [review]

**Comment by zmstone** [comment]
emqx_ctl app ct failed?

**Comment by ieQu1** [comment]
@zmstone 
> emqx_ctl app ct failed?

Indeed, there was a typo. Fixed.

**Review by zmstone — APPROVED** [review]

**Pull request merged**